include/openssl/ocsp.h

```diff
@@ -140,45 +144,52 @@ OPENSSL_EXPORT int OCSP_request_add1_cert(OCSP_REQUEST *req, X509 *cert);
 // includes one or more optional certificates in the request.
 // This will fail if a signature in the |OCSP_REQUEST| already exists.
 OPENSSL_EXPORT int OCSP_request_sign(OCSP_REQUEST *req,
-                      X509 *signer,
-                      EVP_PKEY *key,
-                      const EVP_MD *dgst,
-                      STACK_OF(X509) *certs, unsigned long flags);
+                                     X509 *signer,
+                                     EVP_PKEY *key,
+                                     const EVP_MD *dgst,
+                                     STACK_OF(X509) *certs,
+                                     unsigned long flags);
 
-// Returns response status from |OCSP_RESPONSE|.
+// OCSP_response_status returns response status from |OCSP_RESPONSE|.
 OPENSSL_EXPORT int OCSP_response_status(OCSP_RESPONSE *resp);
 
-// Returns |OCSP_BASICRESP| from |OCSP_RESPONSE|.
+// OCSP_response_get1_basic returns |OCSP_BASICRESP| from |OCSP_RESPONSE|.
 OPENSSL_EXPORT OCSP_BASICRESP *OCSP_response_get1_basic(OCSP_RESPONSE *resp);
 
-// Looks up a cert id and extract the update time and revocation status of
-// certificate sent back from OCSP responder if found. Returns 1 on success.
+// OCSP_resp_find_status looks up a cert id and extract the update time and
+// revocation status of  certificate sent back from OCSP responder if found.
+// Returns 1 on success.
 //
-// Note: 1. Revocation status code is passed into |*status| parameter. Status code will
-//          not be passed if |*status| is NULL.
-OPENSSL_EXPORT int OCSP_resp_find_status(OCSP_BASICRESP *bs, OCSP_CERTID *id, int *status,
-                          int *reason,
-                          ASN1_GENERALIZEDTIME **revtime,
-                          ASN1_GENERALIZEDTIME **thisupd,
-                          ASN1_GENERALIZEDTIME **nextupd);
-
-// Verifies a basic response message. Returns 1 on success, 0 on error, or -1 on
-// fatal errors such as malloc failure.
+// Note: 1. Revocation status code is passed into |*status| parameter. Status
+//          code will not be passed if |*status| is NULL.
+OPENSSL_EXPORT int OCSP_resp_find_status(OCSP_BASICRESP *bs,
+                                         OCSP_CERTID *id,
+                                         int *status,
+                                         int *reason,
+                                         ASN1_GENERALIZEDTIME **revtime,
+                                         ASN1_GENERALIZEDTIME **thisupd,
+                                         ASN1_GENERALIZEDTIME **nextupd);
+
+// OCSP_basic_verify verifies a basic response message. Returns 1 on success,
```

Comment:
Does success == valid? Or just that the function ran to completion and you need to call something like OCSP_is_valid?
```suggestion
// OCSP_basic_verify verifies a basic response message. Returns 1 if the response is valid,
```

Reply:
Yeah, It test's that it's a valid response according to the certs passed in. Next step in a normal OCSP pipeline would to be check that the response hasn't expired.